bug_actiongroup.php

```diff
@@ -206,19 +208,27 @@
 		case 'UP_STATUS':
 			$f_status = gpc_get_int( 'status' );
 			if( access_has_bug_level( access_get_status_threshold( $f_status, $t_bug->project_id ), $t_bug_id ) ) {
-				if( true == bug_check_workflow( $t_status, $f_status ) ) {
+				if( true == bug_check_workflow( $t_status, $f_status ) ||
+					access_compare_level( current_user_get_access_level(), config_get_access( 'status_enum_workflow' ) ) ) {
```

Comment:
Is this access check different than the one in the if statement above?

Reply:
Other user workflow experience is being tested on an individual issue page.

I believe the group status change feature is more likely to be used by an experienced developer, let alone a person having privileges to change workflow should not get confused here.

Reply:
> Wouldn't that confuse the person who is setting the workflow or not allow them to test what other users would experience?

Right, but @vboctor are you aware that there is no checking when using SOAP/REST API?
This can be even more confusing.

> I would rather not have a different there.

👍 at least there will be no confusion if just the UI is used for changes.

In future, all high level checks could be removed if the the workflow check takes place in function validate in a new command IssueStatusChangeCommand.